.kokoro/common.cfg

```diff
@@ -29,3 +29,8 @@ env_vars: {
     value: "gcr.io/cloud-devrel-kokoro-resources/java"
 }
 
+action {
+  define_artifacts {
+    regex: "**/*sponge_log.xml"
+  }
+}
```

Comment:
Les may prefer an empty line at EOF.

Reply:
I have one in my local commit. If there wasn't one wouldn't there be a little icon here indicating that there isn't?

Reply:
Not sure - I can see one on the last commit, but I can't see one here. I'm not sure if it's a big deal, I just know Les has asked me to change it before.

Reply:
Judging from other commits, when there isn't a newline on a file, it has a little red "no" icon on the last line. Combine that with the newline on my local version that is up to date, I think we are ok.